optimum/exporters/tasks.py

```diff
@@ -239,6 +239,10 @@ class TasksManager:
             "default",
             onnx="CLIPOnnxConfig",
         ),
+        "cliptext": supported_tasks_mapping(
```

Comment:
how different is it from CLIP?

Reply:
This was added to export a `CLIPTextModel`  (only composed of the text encoder)

Reply:
I see thanks